srcpkgs/seatd/template

```diff
@@ -0,0 +1,36 @@
+# Template file	for	'seatd'
+pkgname=seatd
+version=0.2.0
+revision=1
+build_style=meson
+configure_args="-Dexamples=disabled -Dlogind=enabled"
+hostmakedepends="pkg-config	scdoc"
```

Comment:
I think there's a tab instead of a space there.

Reply:
heh, I think I may have accidentally replaced all spaces with tabs instead of just the indentation ones.